### ISSUE TITLE
local.d/*.ini does not take effect

### ISSUE THREAD

**Issue by ybian** [klaemo/docker-couchdb#66]
Thanks for your good job. These docker images help us a lot. 

I met a problem trying to use local.d configuration. It does not work as in document: [couchdb config](http://docs.couchdb.org/en/latest/config/intro.html)
I'm using `2.0-rc2` image via the following command:

```
$ docker run -it -v /home/ybian/local.d:/opt/couchdb/etc/local.d -p 5984:5984 klaemo/couchdb:2.0-rc2
```

There is one *.ini file under my local folder `/home/ybian/local.d`, which has some CORS and admin party configuration. But it does not take effect. Seems that couchdb does not read that file at all. I checked the permission so that should not be the culprit. The directory got mounted successfully.

Is this a couchdb issue or image issue or my issue?

**Comment by ybian** [comment]
Confirmed that this is a couchdb 2.0 bug: https://issues.apache.org/jira/browse/COUCHDB-3089

**Comment by klaemo** [comment]
Thanks for investigating and following up on this :)

**Comment by klaemo** [comment]
Should work now with the newly released `2.0-rc4`
